hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -2324,7 +2324,10 @@
             - For HardwareRevisionSecret, use the constant values in design.
             - Configure the keymgr and advance to `CreatorRootKey` and `OwnerIntermediateKey`.
             - Check keymgr internal keys after advance operations.
-            - Generate identity / SW output for both Attestation CDI and Sealing CDI.
+            - Generate identity / SW output for the Sealing CDI.
+               - No need to test the Attestation CDI in chip-level as the only difference is to
+                 use another set of CSR values, and the rest of inputs are the same as the Sealing
+                 CDI.
```

Comment:
yeah, if the delta is an input from another block, it's worth testing it in chip-level as part of the integration test. Since the delta is on the CSRs, probably we can rely on block-level to test it. WDYT?